package.json

```diff
@@ -48,9 +48,14 @@
   "comments": {
     "reason": "Removed from devDependencies as nowhere referenced, commented as reminder",
     "core-js": "^3.31.1",
+    "eslint-config-standard": "^17.0.0",
```

Comment:
Can these packages be dropped then?

Reply:
This is just a comment section for ease of playing around. They are not used by npm install.
I will drop all of them in a forseeable timeframe if we do not get issues.